Tests/iaas/flavor-naming/flavor-names-openstack.py

```diff
@@ -58,7 +58,7 @@ def main(argv):
                                        ("os-cloud=", "mand=", "verbose", "help", "quiet", "v2plus",
                                         "v3", "v1prefer", "accept-old-mandatory"))
     except getopt.GetoptError as exc:
-        print(f"{exc}", file=sys.stderr)
+        print(f"CRITICAL: {exc!r}", file=sys.stderr)
```

Comment:
For the future: we should use some logging module instead of print.

Reply:
I agree, but I wanted to deal with this separately.